spyder/plugins/switcher/widgets/switcher.py

```diff
@@ -0,0 +1,472 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Switcher Main Widget."""
+
+# Standard library imports
+import os
+import sys
+
+# Third party imports
+from qtpy.QtCore import (QEvent, QObject, QSize, QSortFilterProxyModel, Qt,
+                         Signal, Slot, QModelIndex)
+from qtpy.QtGui import QStandardItem, QStandardItemModel, QTextDocument
+from qtpy.QtWidgets import (QAbstractItemView, QApplication, QDialog,
+                            QLineEdit, QListView, QListWidgetItem, QStyle,
+                            QVBoxLayout)
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.manager import CONF
+from spyder.config.utils import is_ubuntu
+from spyder.plugins.switcher.widgets.proxymodel import SwitcherProxyModel
+from spyder.plugins.switcher.widgets.switcheritem import SwitcherItem, SwitcherBaseItem, SwitcherSeparatorItem
+from spyder.py3compat import TEXT_TYPES, to_text_string
+from spyder.utils.icon_manager import ima
+from spyder.widgets.helperwidgets import HTMLDelegate
+from spyder.utils.stringmatching import get_search_scores
+from spyder.plugins.switcher.utils import clean_string
+
+# Style dict constants
+FONT_SIZE = 10
+ITEM_STYLES = {
+        'title_color': ima.MAIN_FG_COLOR,
+        'description_color': 'rgb(153, 153, 153)',
+        'section_color': 'rgb(70, 179, 239)',
+        'shortcut_color': 'rgb(153, 153, 153)',
+        'title_font_size': FONT_SIZE,
+        'description_font_size': FONT_SIZE,
+        'section_font_size': FONT_SIZE,
+        'shortcut_font_size': FONT_SIZE,
+    }
+ITEM_SEPARATOR_STYLES = {
+        'color': ima.MAIN_FG_COLOR,
+        'font_size': FONT_SIZE,
+    }
```

Comment:
```suggestion
    'title_color': ima.MAIN_FG_COLOR,
    'description_color': 'rgb(153, 153, 153)',
    'section_color': 'rgb(70, 179, 239)',
    'shortcut_color': 'rgb(153, 153, 153)',
    'title_font_size': FONT_SIZE,
    'description_font_size': FONT_SIZE,
    'section_font_size': FONT_SIZE,
    'shortcut_font_size': FONT_SIZE,
}

ITEM_SEPARATOR_STYLES = {
    'color': ima.MAIN_FG_COLOR,
    'font_size': FONT_SIZE,
}
```

The lines inside these dicts have more indentation than necessary.